jupyterlab/labextensions.py

```diff
@@ -332,22 +340,30 @@ def run_task(self):
 
 class EnableLabExtensionsApp(BaseExtensionApp):
     description = "Enable labextension(s) by name"
+    aliases = enable_aliases
+
+    user = Bool(False, config=True, help="Whether to do a user install")
+    sys_prefix = Bool(True, config=True, help="Use the sys.prefix as the prefix")
```

Comment:
On the surface, these two very different wordings for the help make it seem like these options control very different things (this confusion would also be solved if we had a single level parameter)